ee/tabby-webserver/src/schema/mod.rs

```diff
@@ -399,11 +399,17 @@ impl Mutation {
         ))
     }
 
-    async fn create_repository(ctx: &Context, name: String, git_url: String) -> Result<ID> {
+    async fn create_repository(
+        ctx: &Context,
+        name: String,
+        git_url: String,
+    ) -> Result<ID, RepositoryError> {
+        let repository = CreateRepositoryInput { name, git_url };
```

Comment:
validation should happens at service level, instead of API route level.

See https://github.com/TabbyML/tabby/blob/4015615cddd21f8a630cbe92c028cb8adbff9c03/ee/tabby-webserver/src/schema/auth.rs#L354